modulus/models/dlwp_healpix_layers/healpix_blocks.py

```diff
@@ -493,6 +493,54 @@ def forward(self, x):
         # return second convnext result
         return self.skip_module2(x1) + self.convblock2(x1)
 
+class Multi_SymmetricConvNeXtBlock(th.nn.Module):
+    """
+    Support "n_layers" in SymmetricConvNeXtBlock.
+    Create a multi-layer SymmetricConvNeXtBlock with 'n_layers' specified in the config files
+    Introduce a new class to prevent conflicts with existing trained models.
+    """
+    def __init__(
+            self,
+            geometry_layer: th.nn.Module = HEALPixLayer,
+            in_channels: int = 3,
+            latent_channels: int = 1,
+            out_channels: int = 1,
+            kernel_size: int = 3,
+            dilation: int = 1,
+            upscale_factor: int = 4,
+            n_layers: int = 1,
```

Comment:
This all looks good to me. Minor point - why do we use 'n_layers' instead of 'n_blocks'? 'n_layers' seems like an over-loaded term since it's also used as an argument to SymmetricConvNeXtBlock.

Reply:
Thanks. For clarity, I replaced 'n_layers' with 'n_conv_blocks'.